File: src/Dependencies/libtase2-2.3.0/jna/src/main/com/libtase2/server/SentPointValue.java
package com.libtase2.server;

/*
 *  Copyright 2021 MZ Automation GmbH
 *
 *  This file is part of libtase2
 */

import com.libtase2.common.PointValue;

public class SentPointValue {
    private DataPoint dataPoint;
    private PointValue pointValue = null;

    SentPointValue(DataPoint dataPoint, PointValue pointValue) {
        this.dataPoint = dataPoint;
        this.pointValue = pointValue;
    }

    public DataPoint getDataPoint() {
        return this.dataPoint;
    }

    public PointValue getPointValue() {
        return this.pointValue;
    }
}

File: src/Dependencies/libtase2-2.3.0/NET/libtase2.NET/server/SentPointValue.cs
using System;
using TASE2.Library.Common;
using TASE2.Library.Server;

namespace libtase2.NET.server
{
    public class SentPointValue
    {
        private DataPoint dataPoint = null;
        private PointValue pointValue = null;

        internal SentPointValue(DataPoint dataPoint, PointValue pointValue)
        {
            this.dataPoint = dataPoint;
            this.pointValue = pointValue;
        }

        public DataPoint DataPoint
        {
            get
            {
                return this.dataPoint;
            }
        }

        public PointValue PointValue
        {
            get
            {
                return this.pointValue;
            }
        }
    }
}
